Makefile

```diff
@@ -286,10 +286,10 @@ prepare-release-branch: yq kustomize ## Prepare the release branch with the rele
 ##@ Debug
 
 # Build an image that can be used with kubectl debug
-# Developers don't need to build this image, as it will be available as gcr.io/k8s-staging-kueue/debug
+# Developers don't need to build this image, as it will be available as us-central1-docker.pkg.dev/k8s-staging-images/kueue/debug
 .PHONY: debug-image-push
 debug-image-push: ## Build and push the debug image to the registry
-	$(IMAGE_BUILD_CMD) -t $(IMAGE_REGISTRY)/debug:$(GIT_TAG) \
+	$(IMAGE_BUILD_CMD) -t $(IMAGE_REPO)/debug:$(GIT_TAG) \
```

Comment:
Is this change `IMAGE_REGISTRY` -> `IMAGE_REPO` desired? IIUC this seems to result in `us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue/debug`

Reply:
Right. Reverted, thanks.